enterprise/cluster/src/main/java/org/neo4j/cluster/client/Clusters.java

```diff
@@ -203,14 +201,7 @@ public boolean isFullHaMember()
 
         private static String localhost()
         {
-            try
-            {
-                return InetAddress.getLocalHost().getHostAddress();
-            }
-            catch ( UnknownHostException e )
-            {
-                throw new RuntimeException( e );
-            }
+            return "127.0.0.1";
```

Comment:
Why do we need this particular address? Shouldn't it be sufficient to have a consistent string returned? If that's the case, then why do we need to change this code, since it seems to be returning a consistent string anyway? I fear this may be masking a problem where somewhere else in the network stack we are making a different assumption about our available interfaces. That doesn't necessarily mean that this change is wrong, but we may be missing a deeper bug.